server/src/Data/Data.hpp

```diff
@@ -0,0 +1,11 @@
+#ifndef DATA_HPP
+#define DATA_HPP
+
+struct Data
+{
+    //some data from application
```

Comment:
let it be just `std::string currentCO2Level`

Reply:
I wanted you to create variable here

Reply:
Done